node_follower/src/lib.rs

```diff
@@ -0,0 +1,58 @@
+pub mod error;
+pub mod gateway_resp;
+use helium_proto::services::{follower, Channel, Endpoint};
+use http::Uri;
+use std::{env, str::FromStr, time::Duration};
+use tonic::Streaming;
+
+pub use error::{Error, Result};
+
+const CONNECT_TIMEOUT: Duration = Duration::from_secs(5);
+const RPC_TIMEOUT: Duration = Duration::from_secs(5);
+const DEFAULT_URI: &str = "http://127.0.0.1:8080";
+
+type FollowerClient = follower::Client<Channel>;
+
+#[derive(Debug, Clone)]
+pub struct FollowerService {
+    pub client: FollowerClient,
+}
```

Comment:
```suggestion
#[derive(Debug, Clone)]
pub struct FollowerClient {
    pub client: follower::Client<Channel>,
}
```
especially now that this is being abstracted behind a sub-crate i think it would be helpful to reaffirm this is just a client for a "remote" grpc server and it doesn't really operate like a "service" in the sense of a looping thread/process?